### ISSUE TITLE
Improving Coding for Everyone

### ISSUE THREAD

**Issue by rebeccaradding** [foundersandcoders/nazareth-programme#184]
There has been much discussion around creating more structure at C4E, including:

- Advertising the opportunity to work on specific projects
- Creating 4 week cycles where people sign up to attend, commit to a project, and pair on it for a month
- Creating guidance around facilitating pair programming at C4E

See ongoing discussion in London: https://github.com/foundersandcoders/selection-committee/issues/58

Would anyone be interested in thinking about this together?

**Comment by mattlub** [comment]
I still think this is relevant

**Comment by rebeccaradding** [comment]
@mattlub I agree, but I noticed that @leemordechai had opened #244 so I closed my issue as I figured it was out of date/a duplication.

**Comment by sofer** [comment]
@mattlub @rebeccaradding do we know how we can improve the sessions? Is it just about the guidelines (#244) or is it other things like student turnout or mentor attendance or relevance of the material?

**Comment by rebeccaradding** [comment]
@sofer we may also want to approach our partners about a subscription to meetup.com to allow for consistent advertisement.
